cmd/minishift/cmd/profile/profile_delete.go

```diff
@@ -69,7 +67,8 @@ func runProfileDelete(cmd *cobra.Command, args []string) {
 		}
 	}
 
-	api := libmachine.NewClient(profileBaseDir, constants.MakeMiniPath("certs"))
+	profileDirs := cmdState.NewMinishiftDirs(constants.GetProfileHomeDir(profileName))
```

Comment:
@LalatenduMohanty `NewMinishiftDirs` is something reminds that we are doing a contractor operation here like creating those directories but in reality, we are just getting the respective directory Name under the profile. I think we should call it as `GetMinishiftDirs` instead `NewMinishiftDirs`

Reply:
@LalatenduMohanty Sure that would do also.